### DISCUSSION TITLE
Background Jobs with v5

### DISCUSSION THREAD

**Discussion by yarapolana** [adonisjs/core#1653] [Help] [answered]
Hey there,
New to the queue world, has anyone managed to set up background jobs in Adonis v5?
In v4 I used to use a library called Adonis-kue, and tried working around it but was unsuccessful.
Has anyone tried creating their own provider in v5? Thinking of doing something with bull but no idea where to start.

**Comment by yarapolana** [comment, marked as answer]
Closing this since I found out the `Mail.sendLater` method and that was exactly what I wanted to do

**Comment by jpedroschmitz** [comment]
Hey, I'm a maintainer from [@rocketseat/adonis-bull](https://github.com/Rocketseat/adonis-bull) and we have released the next version of the library, which is ready for Adonis v5 and uses `bullmq` under the hood for background jobs 🚀 

You can find more information about it [here](https://github.com/Rocketseat/adonis-bull/tree/alpha#readme) 💜

**Reply by yarapolana** [reply]
@jpedroschmitz Is the library being maintained? I see the latest release is alpha.

**Reply by jpedroschmitz** [reply]
@yarapolana I'm no longer working at Rocketseat, and unfortunately, I don't have time to work on this library. Sorry :(
